.travis.yml

```diff
@@ -111,6 +111,52 @@ matrix:
         before_script: skip
         after_script: skip
 
+    - name: Robot Framework Tests
+      env:
+        - TEST_GROUP=acceptance
+      php: "7.4"
+      dist: xenial
+      install:
+        - chmod -R a+x config/Shared/ci/travis/
+        - composer install --optimize-autoloader --no-interaction
+        - config/Shared/ci/travis/install_elasticsearch.sh
+        - config/Shared/ci/travis/install_mailcatcher.sh
+        - >
+          echo "
+          # This file is required by python's package manager PIP
+          # Spryker uses this to run Robot tests in CI
+
+          robotframework==3.2.1
+          robotframework-seleniumlibrary==4.4.0
+          robotframework-seleniumtestability==1.1.0
+          " > requirements.txt
+        - cat requirements.txt
+        - pip3 install --user pipenv
+        - pipenv --python 3 install -r requirements.txt
+        - git clone git@github.com:spryker/robotframework-suite-tests.git --branch b2b --single-branch --depth 1 robotframework-tests
```

Comment:
`b2b` is the target branch, it's not ideal for idempotent testing because this is just whatever commit is latest on that branch. However, this is good for developers who just want to change tests and re-start travis to see results.